frontend/src/services/graphQL.ts

```diff
@@ -32,14 +35,57 @@ export async function graphQLRequest(query: String, variables: ILookup<any> = {}
   return await axios.request(request)
 }
 
+export async function graphQLRequestWithErrorHandling(query: String, variables: ILookup<any> = {}, parse : any = undefined) {
```

Comment:
I don't see what is calling this method? Maybe I missed it?

Reply:
I was going to use it but turned into too much code change for this release.  I think it would be good to start converting the calls over to this in the future to reduce code duplication.

Reply:
No problem. Maybe just leave it commented out with a note? I'd hate to have orphaned coded added if it's not used.